README.md

```diff
@@ -1,6 +1,6 @@
 # Transmorpher Media Server
 
-A media server for images and videos.
+A media server for images, pdfs and videos.
```

Comment:
pdfs or documents? theres a discrepancy to the code, but it might be intented as it only supports pdfs right now.

Reply:
We are talking about pdfs in the readme since it's the only document type supported right now. In the code it's `Document` everywhere, so we don't have to change it later on.